src/SwedbankPay/Api/Client/Version.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SwedbankPay\Api\Client;
+
+/**
+ * Class Version
+ * @package SwedbankPay\Api\Client
+ */
+class Version
+{
+    /**
+     * Gets the version number from a defined constant VERSION, the environment
+     * variable VERSION or from composer.json.
+     *
+     * @return string Version number
+     * @throws ClientException
+     * @SuppressWarnings(PHPMD.ElseExpression)
+     */
+    public static function getVersion()
+    {
+        if (defined('VERSION')) {
+            return VERSION;
+        }
+
+        if (getenv("VERSION") !== false) {
+            return getenv("VERSION");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function getenv() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5251412)